### DISCUSSION TITLE
Prevent Author from editing articles after a certain transition or workflow stage.

### DISCUSSION THREAD

**Discussion by adankupapa** [joomla/joomla-cms#41208] [2. Cold Features]
Just realised it is currently not possible to prevent a usergroup from editing articles after a stage or transition. Would be a great feature to have a transition also change the article's Access Level from **Public to Special** or another existing view level to prevent the article's author from editing the article once the approval process begins.  

**Scenario.** 
1. Lets say we have three stages in our workflow namely Draft Stage, Proof Reading and Publishing. 

2. Author 1 begins an article but is not ready to submit for proof reading so leaves the article in the Draft Stage. As the author has _Edit Own Permission_,  he is able to edit and save that article for as long as it has not yet been transitioned to the next stage.

3. When the author finaly transitions the article to the next stage, It should be possible to configure that transition to not allow further editing from the author. One way I think this can be done is to include an extra field under the **Transition Actions** tab to change the access level to special or any other one. 

Currently, this is not possible. To deny an author editing privileges through the Article permissions tab also denies them access to their unfinished drafts. It means once an an article is saved, author can't touch it even if more details must be added.

**Comment by brianteeman** [comment]
Public and special are viewlevels not permissions(access levels)

**Reply by adankupapa** [reply]
Edited.
